sample-operators/mysql-schema/src/test/java/io/javaoperatorsdk/operator/sample/MySQLSchemaOperatorE2E.java

```diff
@@ -70,9 +70,10 @@ boolean isLocal() {
                     c.replaceDependentResourceConfig(
                         SchemaDependentResource.class,
                         new ResourcePollerConfig(
-                            700, new MySQLDbConfig("127.0.0.1", "3306", "root", "password")));
+                            700, new MySQLDbConfig("127.0.0.1", "3307", "root", "password")));
```

Comment:
Why did you change this to 3307?

Reply:
Sometime 3306 is already taken, but I don't know if it's because i'm working on WSL with vscode and there is an automatic port forwarding. What I would really want is to have a random port generated by the operator extension and have it injected in the conf (like it is often done with wiremock)

Reply:
That is the default mysql port, so usually should not be taken. Otherwise this PR looks good to me.